src/pages/patientView/mutation/MutationInformationContainer.tsx

```diff
@@ -29,6 +29,12 @@ export default class MutationInformationContainer extends React.Component<IMutat
         this.mergedMutations = this.mergeMutations(this.props.mutations);
     }
 
+    componentWillUpdate(nextProps:IMutationInformationContainerProps, nextState:{}) {
+        if (nextProps.mutations !== this.props.mutations) {
+            this.mergedMutations = this.mergeMutations(nextProps.mutations);
```

Comment:
into the MutationTable, we pass in an array of arrays, where each subarray is different instances of the same mutation (i.e. on different samples).

mergeMutations groups together the individual mutation events into these arrays so that the same mutation events are grouped together.

What comes in is a flat list of mutations, so whenever the props get changed, this needs to change too.